lib/matplotlib/cm.py

```diff
@@ -1,8 +1,13 @@
 """
 Builtin colormaps, colormap handling utilities, and the `ScalarMappable` mixin.
 
-See :doc:`/gallery/color/colormap_reference` for a list of builtin colormaps.
-See :doc:`/tutorials/colors/colormaps` for an in-depth discussion of colormaps.
+  - See :doc:`/gallery/color/colormap_reference` for a list of builtin
```

Comment:
Don't indent bullet points. RestructuredText interprets this as a blockquote. This leads to over-indented bullet points in the HTML.

![grafik](https://user-images.githubusercontent.com/2836374/44572972-af663480-a785-11e8-8cdb-836a6d81ccc1.png)

Reply:
Yeah, you could make this a `seealso::`.

If you switch to `seealso::` the following is just for information: Make sure to write bullet lists exactly like this:

~~~
    Some paragraph:

    - Some long text that is continued on
      the next line.
    - Some other text.

    Normal paragraph again.
~~~

1. No indent of the bullets with respect to the surronding paragraphs (that's fixed now).
2. Continued bulleted lines have to have the text alinged with the text after the bullet (you had extra indentation here.
3. There needs to be an empty line before and after the bullet list.

I know this is a bit picky, but that's how restructered text wants it.